controller/src/main/java/org/jboss/as/controller/ModuleIdentifierUtil.java

```diff
@@ -26,30 +21,39 @@ public final class ModuleIdentifierUtil {
      * @param moduleSpec a module name specification in the form {@code name[:slot]}. Cannot be {@code null}
      * @return the canonical representation. Will not return @{code null}
      *
-     * @deprecated use {@link JBossModulesNameUtil#parseCanonicalModuleIdentifier(String)}
+     * @deprecated use {@link #parseCanonicalModuleIdentifier(String)}
      */
     @Deprecated(forRemoval = true, since = "28.0.0")
     public static String canonicalModuleIdentifier(String moduleSpec) {
         return JBossModulesNameUtil.parseCanonicalModuleIdentifier(moduleSpec);
     }
```

Comment:
There are still usages of this method in WildFly. We cannot remove it yet until we have the new version of this class available in WildFly and have switched the method names there.